charters/DeiCharter.md

```diff
@@ -0,0 +1,35 @@
+# Diversity Equity and Inclusion Committee Charter
+
+Project Jupyter welcomes and encourages participation in our community by people of all backgrounds and identities. We are committed to promoting and sustaining a culture that values mutual respect, tolerance, and learning. We work together as a community to help each other live out these values.
+
+The Diversity Equity & Inclusion (DEI) Committee is a standing committee established under Jupyter’s governance model. The committee reviews its charter annually and may make changes at any time during the year. Substantive changes to the purpose or mission must be submitted to the Executive Council for approval.
+
+## WHY: Purpose 
+
+Why is the committee being created? How does this group contribute to Jupyter? 
+
+The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, core developers, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 
+
+## WHAT: Mission/Areas of Responsibility
+
+What will this committee work on? How will the purpose described above be accomplished?
+
+The DEI standing committee is responsible for: 
+* Propose, execute, and evaluate the impact of DEI initiatives
+* Research, establish, and collect metrics related to DEI
+* Advocate for increased participation of underrepresented groups in the Jupyter community. 
+* Serve as a point of contact for DEI topics for the global Jupyter community
+* Include the topic of DEI in discussions and spaces within Project Jupyter.
+* Recommend and promote training, educational, and motivational materials on DEI 
+* Flag DEI issues or potential issues to the Executive Council (EC) or the Software Steering Council (SSC)
+
+## HOW: Meetings
+* Frequency - this committee should meet at least once every two weeks.
+* The Jupyter Decision-Making Guide will be used to make official decisions. 
+* To ensure active participation, a member who is unable to join for more than two thirds of regularly scheduled weekly meetings in a 90 day period may be asked to step down from the committee.
+
+## HOW: Committee Reports
+* Reports will be submitted to the Executive Council half-yearly on or before the 15th of July and January for the preceding calendar half.
+* Reports should include the following metrics:
```

Comment:
I would replace with "Reports are encouraged to include metrics that the Committee establishes are relevant, informative and help in decision making" or similar, rather than hard-coding any specific metric in this charter.